modules/programs/gnome-terminal.nix

```
@@ -75,13 +98,58 @@ let
 
       visibleName = mkOption {
         type = types.str;
-        description = "The profile name.";
+        description =
+          "The profile name. If null, the profile name will be the same as the nix attribute.";
       };
 
       colors = mkOption {
         default = null;
         type = types.nullOr profileColorsSubModule;
-        description = "The terminal colors, null to use system default.";
+        description = ''
+          The terminal colors, null to use system default.
+
```

```suggestion
</para><para>
```

also everywhere else that you have a paragraph break

---

Thanks, I could not figure out how to do it, and the documentation of docbook is a bit too exhaustive to find stuff quickly in it.